_posts/2024-01-12-owasp-top10-with-abstract-examples-part-1.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: OWASP top ten 2021 explained with non-technical examples, </br>Part 1
+author: petteri.poyhtari
+excerpt: >
+   How could we better make the most typical risks of web applications aware of all project personnel, in order to improve the quality of the entire web infrastructure. Part 1 blog post about OWASP top 10 with physical world analogies.
+tags:
+ - DevSecOps
+ - InfoSec
+ - OWASP
+ - Quality
+ - Security
+ - Software security
+ - Web development
+---
+
+## What is this blog about?
+
+System vulnerabilities are found in increasing numbers. Whether the reason is growing crime or the poor quality of application development, projects should take information security requirements seriously. In order to take information security seriously, it also requires commitment from the project's non-technical staff. This blog post will review the most typical web application security risks using the widely known [Open Worldwide Application Security Project top 10 list](https://owasp.org/Top10/). I try to lower the bar as much as I can so everyone could understand what the OWASP top 10 is really about. Why they are security risks and how can the risks associated with them be reduced or eliminated?
+
+## Current cyber weather
+
+Not surprisingly, cyber security should be taken more seriously now than ever. Statistics show (e.g. a report by [Google's threat analysis group](https://security.googleblog.com/2023/07/the-ups-and-downs-of-0-days-year-in.html)) that the number of zero-day vulnerabilities is growing exponentially. Also, the use of 0-day in August 2023 exceeds the figure for the entire year of 2022.
+
+![Infra in cyber weather](/img/2024-01-owasp-top-10/0_cyber_weather_2.jpg)
+
+Does this say more about the global cyber situation or the bad code we produce? Who is to blame here? Juhani Eronen, the leading expert at Traficom's Cybersecurity Center [stated](https://www.kyberturvallisuuskeskus.fi/fi/ajankohtaista/miten-ohjelmistokehityksen-turvallisuutta-voidaan-kehittaa-tuore-selvitys-kartoitti), "*According to our observations, poor software quality is one of the biggest security problems.*" 
```

Comment:
Maybe very briefly state what Traficom is, for non Finnish audience, when mentioning it the first time?

Reply:
Good point. I'll do that.

Reply:
Oisko nyt hyvä? Lisäsin tiivistetyn maininnan + linkit for furhter information.